.bumpversion.cfg

```diff
@@ -0,0 +1,33 @@
+[bumpversion]
```

Comment:
Question ahead of time: "Can I add comments about what each section does?" 
Answer: `bump2version` de-setializes this file, then re-serializes this file after the bump is performed to re-write `current_version` (That value needs to remain "in sync" with the logical version), since comments are not serializable, they get dropped.

Reply:
Do we have notes anywhere on how to install bump2version? Would be good to document in the repo